editor/editor_inspector.cpp

```diff
@@ -3323,6 +3338,14 @@ void EditorInspector::_property_deleted(const String &p_path) {
 		return;
 	}
 
+	if (p_path.begins_with("metadata/")) {
+		String name = p_path.replace_first("metadata/", "");
+		undo_redo->create_action("Remove metadata");
```

Comment:
```suggestion
		undo_redo->create_action(vformat(TTR("Remove metadata %s"), name));
```
I bet Calinou would change this in some future PR, so could as well be done now.